Calculator.Tests/CalculatorTests.cs

```diff
@@ -0,0 +1,33 @@
+using NUnit.Framework;
+
+namespace Calculator.Tests
+{
+    public class CalculatorTests
```

Comment:
You have to write boundary tests:
 - what happens when you ad 0 with 0
 - what happens when you add max(int) + max(int)
 - what happens when you add -max(int) + -max(int)